aspnetcore/migration/proper-to-2x/index.md

```diff
@@ -1,211 +1,47 @@
 ms.author: riande
 ms.date: 10/18/2019
 uid: migration/proper-to-2x/index
 ---
-# Migrate from ASP.NET to ASP.NET Core
+# Migrate from ASP.NET Framework to ASP.NET Core
 
 By [Isaac Levin](https://isaaclevin.com)
 
  :::moniker range=">= aspnetcore-7.0"
 
-This article serves as a reference guide for migrating ASP.NET Framework apps to ASP.NET Core.
+Most non-trivial ASP.NET Framework apps should consider using the [incremental migration](xref:migration/inc/overview) approach.
 
-Visual Studio has tooling to help migrate ASP.NET apps to ASP.NET Core. For more information, see [Migrating from ASP.NET to ASP.NET Core in Visual Studio](https://devblogs.microsoft.com/dotnet/introducing-project-migrations-visual-studio-extension/).
+The ebook [Porting existing ASP.NET apps to .NET Core](https://aka.ms/aspnet-porting-ebook) contains information:
 
-The [.NET Upgrade Assistant](https://dotnet.microsoft.com/en-us/platform/upgrade-assistant) is a command-line tool that can help migrate ASP.NET to ASP.NET Core. For more information, see [Overview of the .NET Upgrade Assistant](/dotnet/architecture/porting-existing-aspnet-apps/) and [Upgrade an ASP.NET MVC app to .NET 6 with the .NET Upgrade Assistant](/dotnet/core/porting/upgrade-assistant-aspnetmvc).
+* On porting ASP.NET Framework apps to ASP.NET Core.
+* Incremental migration information that is outdated. See <xref:migration/inc/overview> for the definitive guide to incremental migration.
 
-See the ebook [Porting existing ASP.NET apps to .NET Core](https://aka.ms/aspnet-porting-ebook) for a comprehensive porting guide.
+## URI decoding differences between ASP.NET to ASP.NET Core
 
-## Prerequisites
+ASP.NET Core has the following URI decoding differences with ASP.NET Framework:
 
-[.NET Core SDK 2.2 or later](https://dotnet.microsoft.com/download)
+| ASCII   | Encoded | ASP.NET Core | ASP.NET Framework |
+| ------------- | ------------- | ------------- | ------------- |
+| `\` | `%5C`  |  `/` |  `\` |
+| `/` | `%2F`  |  `%2F` |  `/` |
 
-## Target frameworks
+When decoding `%2F` on ASP.NET Core:
 
-ASP.NET Core projects offer developers the flexibility of targeting .NET Core, .NET Framework, or both. See [Choosing between .NET Core and .NET Framework for server apps](/dotnet/standard/choosing-core-framework-server) to determine which target framework is most appropriate.
+* The entire path gets unescaped except `%2F` because converting it to `/` would change the path structure. It can’t be decoded until the path is split into segments.
+* See the [PathBase source](https://source.dot.net/#Microsoft.AspNetCore.Http.Abstractions/HttpRequest.cs,8d85f458c32cb4a5) for more information.
 
-When targeting .NET Framework, projects need to reference individual NuGet packages.
+To generate the value for `HttpRequest.Url`, use `new Uri(this.AspNetCoreHttpRequest.GetEncodedUrl());` to avoid `Uri` misinterpreting the values.
```

Comment:
@Tratcher please review. This section fixes #22352 and doesn't seem to fit in the incremental docs.

Reply:
:shipit: